force-app/main/default/pages/SummitEventsRegister.page

```diff
@@ -530,7 +530,7 @@ Created by Thaddaeus Dahlberg on 5/1/2018.
                                         </apex:outputText>
                                     </apex:outputLabel>
                                     <div class="slds-form-element__control">
-                                        <apex:inputText value="{!newEvtReg.Registrant_Street_1__c}" id="street" required="true" styleClass="slds-input" disabled="{!contactSet}" maxLength="95"/>
+                                        <apex:inputText value="{!newEvtReg.Registrant_Street_1__c}" id="street" required="{!IF(CONTAINS(eventPage.Ask_Mailing_Address__c, 'require'),true,false)}" styleClass="slds-input" disabled="{!contactSet}" maxLength="95"/>
```

Comment:
{!IF(CONTAINS(eventPage.Ask_Mailing_Address__c, 'require'),true,false)}
can be simplified to:
CONTAINS(eventPage.Ask_Mailing_Address__c, 'require')

Reply:
This is true. I tended to keep the code more verbose because I would copy and paste the same logic and text or style changes were in place of true and false. Just being lazy :)